src/Sulu/Component/Webspace/Analyzer/Attributes/WebsiteRequestProcessor.php

```diff
@@ -70,6 +86,11 @@ function (PortalInformation $a, PortalInformation $b) {
 
         /** @var PortalInformation $portalInformation */
         $portalInformation = reset($portalInformations);
+        foreach ($portalInformations as $information) {
```

Comment:
ok, you are already using that variable name... What about renaming the other variable to be more clear what it is? Something like `defaultPortalInformation` or `currentPortalInformation`? Just guessing, I actually don't even really know what you are doing here 😉